ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectra.java

```diff
@@ -31,6 +31,30 @@ static SpecConfigElectra required(final SpecConfig specConfig) {
                         + specConfig.getClass().getSimpleName()));
   }
 
+  UInt64 getMinActivationBalance();
+
+  UInt64 getMaxEffectiveBalanceElectra();
+
+  UInt64 getPendingBalanceDepositsLimit();
+
+  UInt64 getPendingPartialWithdrawalsLimit();
+
+  UInt64 getPendingConsolidationsLimit();
+
+  int getWhistleblowerRewardQuotientElectra();
+
+  int getMinSlashingPenaltyQuotientElectra();
+
+  int getMaxAttesterSlashingsElectra();
+
+  int getMaxAttestationsElectra();
+
+  int getMaxConsolidations();
+
+  int getMaxPartialWithdrawalsPerPayload();
+
+  UInt64 getMinPerEpochChurnLimitElectra();
```

Comment:
Are we missing `MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT`?

https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L168C1-L168C42

Reply:
it does look like thats needed, will add it! good catch.